airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2692,6 +2692,22 @@ components:
       properties:
         sourceId:
           $ref: "#/components/schemas/SourceId"
+    SourceCloneRequestBody:
```

Comment:
can you add a description field for `SourceCloneRequestBody` and `DestinationCloneRequestBody`?

Reply:
Will do.